apps/neuvector/patching-crds1/kustomize.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: neuvector-crds
+  namespace: flux-system
+spec:
+  interval: 10m
+  path: ./apps/neuvector/patching-crds
```

Comment:
You've called your directory `patching-crds1` so this wont currently find it.
You should be able to call your directory `patching-crds` and just uncomment the existing commented lines to use it.